### ISSUE TITLE
Dependency conflict with firebase plugins on `play-services-base`

### ISSUE THREAD

**Issue by daniel-v** [loup-v/geolocation#4]
Current firebase plugins depend on 11.8.0 while geolocation on 12.0.1. Don't know what the resolution is and not a big mobile developer. Tips how this could be resolved?

**Comment by daniel-v** [comment]
Temporary workaround in `app/build.gradle`

```
configurations.all {
    resolutionStrategy.eachDependency { DependencyResolveDetails details ->
        if (details.requested.group == 'com.google.android.gms' && details.requested.name == 'play-services-location') {
            details.useVersion '11.8.0'
        }
    }
}
```

**Comment by lukaspili** [comment]
Thanks for the report. I will match the plugin's firebase version to other firebase plugins.

**Comment by lukaspili** [comment]
@flutterite use this:

```groovy
configurations.all {
    resolutionStrategy {
        eachDependency { details ->
            if (details.requested.group == 'com.google.android.gms' ||
                    details.requested.group == 'com.google.firebase') {
                details.useVersion "11.8.0"
            }
        }
    }
}
```

**Comment by lukaspili** [comment]
Fixed in 0.2.1.
Matched version to Flutter's firebase plugins.
